styles.css

```diff
@@ -33,23 +33,64 @@ body {
 }
 
 .hamburguer {
+  position: relative;
   margin-right: 36px;
   height: 12px;
   width: 18px;
+  z-index: 10;
 }
 
 .bar {
   display: block;
-  background-color: #6070FF;
+  background-color: #6070ff;
   width: 20px;
   height: 2px;
   margin: 2px auto;
   -webkit-transition: all 0.3s ease-in-out;
   transition: all 0.3s ease-in-out;
 }
 
+.hamburguer.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburguer.active .bar:nth-child(1) {
+  transform: translateY(4px) rotate(45deg);
+  background-color: #fff;
+}
+
+.hamburguer.active .bar:nth-child(3) {
+  transform: translateY(-4px) rotate(-45deg);
+  background-color: #fff;
+}
+
 .nav_menu {
-  display: none;
+  position: fixed;
+  z-index: 0;
+  left: 100%;
+  top: 0;
+  gap: 10%;
+  flex-direction: column;
+  padding-top: 70px;
+  background-color: #6070ff;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(5px);
+  width: 100%;
+  height: 100%;
+  text-align: left;
+  transition: 0.3s;
+}
+
+.nav_li_item {
```

Comment:
Your nav item is too close to each other. Kindly give some space between the nav items like on the Figma template. 👍 
<img width="397" alt="Screenshot 2023-04-10 at 11 42 06 PM" src="https://user-images.githubusercontent.com/25887253/230996141-349e1d63-e1e2-461c-a0a9-403774001345.png">
<img width="397" alt="Screenshot 2023-04-10 at 11 42 51 PM" src="https://user-images.githubusercontent.com/25887253/230996150-4ca5cbfe-6ffb-46f4-b17f-bb9540aa24ee.png">

Reply:
thank you @huseyinbiyik , fixed changes.